tests/posit/connect/test_jobs.py

```diff
@@ -0,0 +1,157 @@
+        content = c.content.get("f2f37341-e21d-3d80-c698-a935ad614066")
+
+        assert len(content.jobs) == 1
+
+
+class TestJobsFind:
+    @responses.activate
+    def test(self):
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json"),
+        )
+
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs/tHawGvHZTosJA2Dx",
+            json=load_mock(
+                "v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs/tHawGvHZTosJA2Dx.json"
+            ),
+        )
+
+        c = Client("https://connect.example", "12345")
+        content = c.content.get("f2f37341-e21d-3d80-c698-a935ad614066")
+
+        job = content.jobs.find("tHawGvHZTosJA2Dx")
+        assert job["key"] == "tHawGvHZTosJA2Dx"
+
+    @responses.activate
+    def test_cached(self):
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json"),
+        )
+
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs.json"),
+        )
+
+        c = Client("https://connect.example", "12345")
+        content = c.content.get("f2f37341-e21d-3d80-c698-a935ad614066")
+
+        assert content.jobs
+        job = content.jobs.find("tHawGvHZTosJA2Dx")
+        assert job["key"] == "tHawGvHZTosJA2Dx"
+
+    @responses.activate
+    def test_miss(self):
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json"),
+        )
+
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs.json"),
+        )
+
+        c = Client("https://connect.example", "12345")
+        content = c.content.get("f2f37341-e21d-3d80-c698-a935ad614066")
+
+        assert content.jobs
+        with pytest.raises(ValueError):
+            content.jobs.find("not-found")
+
+
+class TestJobsFindBy:
+    @responses.activate
+    def test(self):
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json"),
+        )
+
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs",
+            json=load_mock("v1/content/f2f37341-e21d-3d80-c698-a935ad614066/jobs.json"),
+        )
```

Comment:
In cases like this, it'd be helpful to see comments explaining what each of the different mocks are testing (if they test different conditions), since that information isn't available here.